bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/CarouselImpl.java

```diff
@@ -129,6 +134,11 @@ public String getAccessibilityLabel() {
         return accessibilityLabel;
     }
 
+    @Override
+    public boolean getPrependControls() {
```

Comment:
I'd rename it `isControlsPrepended()`

Reply:
adressed in a407f8573ac5905bd5d9573fdc05fca0a2753c36